substrate/bin/node/runtime/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,231 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Genesis Presets for the Kitchensink Runtime
+
+use polkadot_sdk::*;
+
+use crate::{
+	constants::currency::*, frame_support::build_struct_json_patch, AccountId, AssetsConfig,
+	BabeConfig, Balance, BalancesConfig, ElectionsConfig, NominationPoolsConfig,
+	RuntimeGenesisConfig, SessionConfig, SessionKeys, SocietyConfig, StakerStatus, StakingConfig,
+	SudoConfig, TechnicalCommitteeConfig, BABE_GENESIS_EPOCH_CONFIG,
+};
+use alloc::{vec, vec::Vec};
+use pallet_im_online::sr25519::AuthorityId as ImOnlineId;
+use sp_authority_discovery::AuthorityId as AuthorityDiscoveryId;
+use sp_consensus_babe::AuthorityId as BabeId;
+use sp_consensus_beefy::ecdsa_crypto::AuthorityId as BeefyId;
+use sp_consensus_grandpa::AuthorityId as GrandpaId;
+use sp_core::{crypto::get_public_from_string_or_panic, sr25519};
+use sp_genesis_builder::PresetId;
+use sp_keyring::Sr25519Keyring;
+use sp_mixnet::types::AuthorityId as MixnetId;
+use sp_runtime::Perbill;
+
+pub const ENDOWMENT: Balance = 10_000_000 * DOLLARS;
+pub const STASH: Balance = ENDOWMENT / 1000;
+
+pub struct StakingPlaygroundConfig {
+	/// (Validators, Nominators)
+	pub dev_stakers: (u32, u32),
+	pub validator_count: u32,
+	pub minimum_validator_count: u32,
+}
+
+/// The staker type as supplied ot the Staking config.
+pub type Staker = (AccountId, AccountId, Balance, StakerStatus<AccountId>);
+
+/// Helper function to create RuntimeGenesisConfig json patch for testing.
+pub fn kitchen_sink_genesis(
```

Comment:
```suggestion
pub fn kitchensink_genesis(
```

Reply:
I would rename `kitchen_sink` to `kitchensink` everywhere to align naming conventions.

Reply:
c4264338abf66d1478f9258978c7e02e0bcb826f